CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.10.0 (Released 2025-03-04)
+
+BREAKING CHANGES
+
+- transfers: update methods to use the `/accounts/{accountID}`-prefixed endpoints
+
+ADDITIONS
+
+- transfers: add cancellation API endpoints and client methods
+- cards: update `CardDetails` model to use new `CardTransactionStatus` enum
```

Comment:
this should go back into `breaking changes` since it's converting an existing field from `string` to an `enum`